requirements.txt

```diff
@@ -1,3 +1,3 @@
 django-model-utils>=2.0,<4.0.0
 # Additional requirements go here
-faker>=0.7.0,<3.0.0
+faker>=0.8.0
```

Comment:
maybe `>=0.8.0,<4.0.0` to be a bit more future-proof?

Reply:
Yes and No. ;)  
I'm definitely fine with setting a range that includes the current version of Faker.  
However they've jumped by 2 major releases in the last few months.  
3.0.0 was released in December adding support for generating Fakes from multiple locales at once and included the block for accessing the seed method on the instance.  
4.0.0 was released only a couple of weeks later and dropped support for Python 2.7 (Not only on paper, they did remove all future imports and changed several things about string handling)  

Actually I'm pretty confused right now, because the tests for django-scrubber still include Python 2.7 and ran through just fine for my PR.

Reply:
So I narrowed the faker version range down to `>=0.8.0,<5.0.0` because it's apparently still working for 4.0.1 and the Faker project seems to adhere to semVer.